src/framework/mpas_stream_manager.F

```diff
@@ -4869,6 +4871,11 @@ subroutine prewrite_reindex(allFields, streamFields) !{{{
 
         integer :: i, j, threadNum
 
+        character (len=StrKIND), pointer :: packages
+        logical :: active_field
+        integer :: err_level
```

Comment:
For consistency with planned changes to PR #1117, I should probably switch to snakeCase here, too.

Reply:
As in the comment discussion on PR #1118, I'm proposing to keep the snake_case here to make it easier to find and factor-out common package checking code in future.